lib/inferno/apps/cli/execute/console_outputter.rb

```diff
@@ -1,15 +1,15 @@
 require 'pastel'
-require_relative '../../web/serializers/test_run'
-require_relative '../../web/serializers/result'
+require 'tty-spinner'
+require_relative 'json_outputter'
 
 module Inferno
   module CLI
     class Execute
       # @private
-      class ConsoleOutputter
-        COLOR = Pastel.new
+      class ConsoleOutputter < JSONOutputter
```

Comment:
Unless I'm missing something, I don't think it makes sense for this to inherit from JSONOutputter.

Reply:
It inherits `serialize`